test-network/network.config

```diff
@@ -0,0 +1,60 @@
+# -i  
+# default image tag
+IMAGETAG="2.5.4"
+# -cai
+# default ca image tag
+CA_IMAGETAG="1.5.7"
+
+# Using crpto vs CA. default is cryptogen
+CRYPTO="cryptogen"
+# -r
+# max number of retries before giving up
+MAX_RETRY=5
+# -d
+# default for delay between commands
+CLI_DELAY=3
+# -c
+# channel name defaults to "mychannel"
+CHANNEL_NAME="mychannel"
+# -s
+# default database
+DATABASE="leveldb"
+# -org
+# default org 
+ORG=1
+#
+# -ccl
+# chaincode language defaults to "NA"
+CC_SRC_LANGUAGE="go"
+# -ccv
+# Chaincode version
+CC_VERSION="1.0.1"
+# -ccn
+# chaincode name defaults to "NA"
+CC_NAME="asset-transfer-basic-go"
```

Comment:
The doc tutorials use a default name of "basic" so that any language can be used without having to change the instructions for post-deployment steps. You may want to use default name of "basic" for consistency, so that people can toggle between the existing tutorials and this new way.

Reply:
Corrected to "basic"